github.com/maistra/istio-operator/pkg/apis/external/istio/config/v1alpha2/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,2040 @@
+//go:build !ignore_autogenerated
```

Comment:
what is this file about?

Reply:
Ah sorry i didn't realise i checked these in

Reply:
strange, my CI failure is saying:
"ERROR: Some files need to be updated, please run 'make gen' and include any changed files in your PR"
And make gen creates those files, what's going on here :confused:

Reply:
I cleared what seems like rubish from a stray go build run.. but it's still erroring on deep copy methods derived from new struct fields .. i am missing something about the workflow here I believe.